src/OSPSuite.Infrastructure.Import/Core/DataFormat/AbstractColumnsDataFormat.cs

```diff
@@ -47,9 +47,45 @@ private double setParameters(IUnformattedData data, IReadOnlyList<ColumnInfo> co
          ExtractGeneralParameters(keys, data, metaDataCategories, ref totalRank);
          ExtractNonQualifiedHeadings(keys, missingKeys, columnInfos, data, ref totalRank);
          setSecondaryColumnUnit(columnInfos);
+         setDimensionsForMappings(columnInfos);
          return totalRank;
       }
 
+      private void setDimensionsForMappings(IReadOnlyList<ColumnInfo> columnInfos)
+      {
+         foreach (var parameter in _parameters.OfType<MappingDataFormatParameter>())
+         {
+            var mappingColumn = parameter.MappedColumn;
+
+            if (mappingColumn?.Unit == null || mappingColumn.Dimension != null)
+               continue;
+
+            var concreteColumnInfo = columnInfos.FirstOrDefault(x => x.Name == parameter.MappedColumn.Name);
+            //initial settings for fraction dimension
+            if (concreteColumnInfo.DefaultDimension?.Name == Constants.Dimension.FRACTION &&
+                mappingColumn.Unit.ColumnName.IsNullOrEmpty() &&
+                mappingColumn.Unit.SelectedUnit == UnitDescription.InvalidUnit)
+            {
+               mappingColumn.Dimension = concreteColumnInfo.DefaultDimension;
+               mappingColumn.Unit = new UnitDescription(mappingColumn.Dimension.BaseUnit.Name);
```

Comment:
Is this supposed to be the baseUnit or the DefaultUnit? We also have a DEfaultUnitName property directly

Reply:
Right, DefaultUnitName should be the one.